resources/js/components/view/DataImporter.vue

```diff
@@ -0,0 +1,668 @@
+                        state.error = '';
+                        state.validationData = data;
+                        console.log(state.validationData);
+                    })
+                    .catch(e => {
+                        showImportError({
+                            message: e.response.data.error,
+                            data: e.response.data.data,
+                        });
+                    })
+                    .finally(_ => state.validating = false);
+            };
+
+            const upload = _ => {
+                state.uploading = true;
+                const formData = buildFormData();
+
+                importEntityData(formData).then(data => {
+                    state.imported = true;
+                    for(let i = 0; i < data.length; i++) {
+                        store.dispatch('addEntity', data[i]);
+                    }
+                    toast.$toast(t('main.importer.success', {
+                        cnt: data.length
+                    }, data.length), '', {
+                        duration: 2500,
+                        autohide: true,
+                        channel: 'success',
+                        icon: true,
+                        simple: true,
+                    });
+                }).catch(e => {
+                    showImportError({
+                        message: e.response.data.error,
+                        data: e.response.data.data,
+                    });
+                }).finally(_ => {
+                    state.uploading = false;
+                });
+            };
+            const addFile = (file) => {
+                if(file) {
+                    state.files = [file];
+                    readContent();
+                }
+            };
+
+            const storageSettings = {
+                fileContent: props.debug,
+                entitySettings: props.debug,
+                attributeMapping: props.debug,
+            };
+
+            // DATA
+
+            const {
+                value: fileContent,
+                reset: resetFileContent,
+                save: saveFileContent
+            } = useOptionalLocalStorage(
+                storageSettings.fileContent,
+                'importer_file_content', ''
+            );
+
+            const {
+                value: entitySettings,
+                reset: resetEntitySettings
+            } = useOptionalLocalStorage(
+                storageSettings.entitySettings,
+                'importer_entity_settings', {
+                entityType: null,
+                entityName: null,
+                entityParent: null,
+            });
+
+            const {
+                value: attributeSettings,
+                reset: resetAttributeSettings
+            } = useOptionalLocalStorage(
+                storageSettings.attributeMapping,
+                'importer_attribute_mapping', {
+                autoComplete: true,
+                mapping: {},
+            });
```

Comment:
can this be removed?

Reply:
I wouldn't as it is a great tool to populate the fields without having to do all the clicks to get back to the previous state.

Reply:
But this is just for the sake of debugging. I'd not add any non-production logic that might be hard to read/understand and is also a possible error source. We also have to take care of it in case of a framework update. And debugging a specific component is a really rare case.

Reply:
Removed in 2cc4f12b53ce12ee3e8e88324e76ad265bc333f8